beta/src/content/docs/en/api/effector/moduleCompat.md

```diff
@@ -0,0 +1,86 @@
+---
+title: effector/compat
```

Comment:
I think, `description` and `redirectFrom` settings should be also added here, as for other pages in the docs 🤔

Reply:
Description is added ✅ 

But for `redirectFrom` I am not certain. In old docs, this module was described in the description of the installation process — https://effector.dev/docs/introduction/installation/#compatibility. So, we do not have a separate page for it to redirect from.